frontend/talentsearch/src/js/components/Router.tsx

```diff
@@ -74,14 +74,29 @@ const routes = (
     }),
   },
   {
-    path: `${profilePaths.skillsAndExperiences()}/:type`,
+    path: `${profilePaths.skillsAndExperiences()}/:type/create`,
     action: (context) => {
       const experienceType = context.params.type as ExperienceType;
       return {
         component: <ExperienceFormContainer experienceType={experienceType} />,
       };
     },
   },
+  {
+    path: `${profilePaths.skillsAndExperiences()}/:type/:id/edit`,
+    action: (context) => {
+      const experienceType = context.params.type as ExperienceType;
+      const experienceId = context.params.id as string;
+      return {
+        component: (
+          <ExperienceFormContainer
+            experienceType={experienceType}
+            experienceId={experienceId}
+          />
+        ),
+      };
+    },
+  },
```

Comment:
It might be a good idea to introduce a sort of edit=true flag here. It will enable conditional operations that depends on someone editing an experience rather than creating one.

If Edit=True, you could have a different toast display, updated successfully instead of created successfully. 
It could also come in handy for the Delete Experience (separate issue), because then the delete button can be conditionally rendered. So it seems like something that could be of use for multiple tasks. 

Of course, you could implement it another way, rather than the edit=true passed in the route

Reply:
That's a good idea. Thanks for the suggestion.